_posts/2016-06-08-component-relative-paths-in-angular-2.md

```diff
@@ -373,7 +373,27 @@ export class HeaderComponent implements OnInit {
 {% endraw %}
 {% endhighlight %}
 
-> Personally, I like the `import` approach more. Thanks to Soós Gábor for his WebPack expertise and insight here. 
+3) Finally, WebPack developers can load templates and styles at runtime by adding `./` at the beginning of the template,
+styles, and styleUrls properties that reference *component-relative URLS.
+
+
+{% highlight js %}
+{% raw %}
+import { Component } from '@angular/core';
+
+@Component({
+  selector : 'my-app',
+  templateUrl: './header.component.html',
```

Comment:
I'm not familar with webpack at all but at least with angular-cli it's not a requirement to prepend with `./`. So this works just fine https://github.com/thoughtram/angular2-master-class-solutions/blob/master/src/app/contacts-header/contacts-header.component.ts#L6